faq/openfabrics.inc

```diff
@@ -2252,73 +2242,53 @@ shell$ mpirun --mca coll_fca_enable 1 --mca coll_fca_np &lt;N&gt; ...
 /////////////////////////////////////////////////////////////////////////
 $q[] = "Does Open MPI support <code>MXM</code>?";
 $anchor[] = "mxm";
+$a[] = "MXM support is currently deprecated and replaced by <a href=\"#ucx\">UCX</a>.";
```

Comment:
Is it useful to keep the old MXM information somewhere?

I.e., there are users out there with older hardware / software.  What do we do about them?

This might be a larger question about the FAQ -- it's not well structured to handle changing information over time.  ☹️

Reply:
UCX supports everything the old MXM has supported, so I prefer to remove the old documentation and steer users to UCX.